.gitignore

```diff
@@ -87,3 +88,6 @@ screenlog.0
 **/*.xcassets/Secrets/*.dataset
 !**/*.xcassets/Secrets/testSecret.dataset
 .bitrise.secrets.yml
+
+# Ignore all GoogleService-Info.plist files
+**/GoogleService-Info.plist
```

Comment:
We have empty placeholder files of such plists for each apps. Shouldn't we delete them also?